Libbna/CUSTOM-CMS
# #254: Added asterik(*) for the required fields

Steps to test:
* Go to all the forms on the site.
* Check if the required fields input box has * Field required text below it as shown in below image.
![image](https://user-images.githubusercontent.com/87715018/152524284-2f4f92ee-5239-4c38-9712-9ca0955d9590.png)
